src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The Bulldozer is doing some work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The Bulldozer stopped working");
```

Comment:
Similar to the doWork method, the stopWork method message could be more descriptive about the state of the Bulldozer when it stops working.